### ISSUE TITLE
Implement BRLTTY and braille displays which connect via BRLTTY into automatic braille display detection

### ISSUE THREAD

**Issue by Adriani90** [nvaccess/nvda#9098]
### Is your feature request related to a problem? Please describe.
There are lots of users wwho are changing from other Screen readers to NVDA. Especially People who used Cobra or Window Eyes before, they have Braille Displays which are not recognized by NVDA without BRLTTY. For example many Baum and Audiodata Braille Displays can only be used via BRLTTY. But there are also lots of other devices we might not know of. Having a Braille Display at Hand to create a Driver is not always possible. So, brltty as an additional Option would be a good Bridge until coresponding Drivers are created, if any.

### Describe the solution you'd like
BRLTTY should come with NVDA package in such a way that the user can choose it out of the Braille Settings. Braille displays, which are connected through BRLTTY should be detected automatically.

### Describe alternatives you've considered
Assuming the user installed BRLTTY manually, BRLTTY should be automatically detected by NVDA and coresponding braille displays should be detected imediately.

**Comment by Adriani90** [comment]
as an example:
BrailleStar 80 from HandyTech is not included in automatic detection and the Braille Display Alva Abt 3.40 works only with BRLTTY. Due to different Features of the Braille Displays, there are users who might use more than one Braille Display at work or privately for different situations.

**Comment by leonardder** [comment]
> BrailleStar 80 from HandyTech is not included in automatic detection

Wait, really? This should be the case, and I'm pretty sure I tested this device. Are you talking about a braille star that is connected using USB?

**Comment by Adriani90** [comment]
Hello, @leonardder sorry it an issue with the usb Connection. Braillestar 80 works with automatic detection. Sorry for any inconvenience.

**Comment by Adriani90** [comment]
One issue which could come up here is that BRLTTY actually has ist own Braille display detection. So in case we want to implement this, we should specify which Braille Displays should use BRLTTY native detection and which ones should use NVDA's automatic detection.

**Comment by Adriani90** [comment]
So for example Braille Displays which already have a Driver implemented in NVDA should use NVDA's automatic Braille Display detection. I cannot test BRLTTY's automatic detection because I do not have a Braille Display. But it should work properly since Jaws I think relies on it as well.

**Comment by leonardder** [comment]
JAWS doesn't bundle BRLTTY at all and doesn't have auto detection, only for Focus displays. Also, most if not all USB devices rely on a separate LibUSB driver bundled with BRLTTY that drives the native connections to these displays unavailable.
